Src/WitsmlExplorer.Frontend/components/ContentViews/WellsListView.tsx

```diff
@@ -54,6 +58,34 @@ export const WellsListView = (): React.ReactElement => {
     });
   };
 
+  useEffect(() => {
```

Comment:
I suggest moving the refresh wells functionality into a `refreshWells` function in `Panel.tsx`. The `Panel.tsx` should then use the currentSelected from navigationState to infer distinguish between refreshing wells, a single well, or an object.

Reply:
Done.